Update ns-System.Drawing for supported apps (#5232)
dotnet/dotnet-api-docs: includes/drawing.md

> [!CAUTION]

> > - The `System.Drawing` namespace is not recommended for new development, because it's not supported within Windows services, ASP.NET Core, and ASP.NET. Attempting to use `System.Drawing` classes within one of these application types, may result in run-time exceptions and diminished service performance. Recommended alternatives include [ImageSharp](https://github.com/SixLabors/ImageSharp), [SkiaSharp](https://github.com/mono/SkiaSharp), and [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).

Author: ```suggestion
> - The `System.Drawing` namespace is not recommended for new development, because it's not supported within Windows services, ASP.NET Core, and ASP.NET. Attempting to use `System.Drawing` classes within one of these application types may result in run-time exceptions and diminished service performance. Recommended alternatives include [ImageSharp](https://github.com/SixLabors/ImageSharp), [SkiaSharp](https://github.com/mono/SkiaSharp), and [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).
```

> > - The `System.Drawing` namespace is not recommended for new development, because it's not supported within Windows services, ASP.NET Core services, and ASP.NET services. If you attempt to use `System.Drawing` classes within one of these application types, you may get run-time exceptions and diminished service performance. Recommended alternatives include [ImageSharp](https://github.com/SixLabors/ImageSharp), [SkiaSharp](https://github.com/mono/SkiaSharp), and [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).

Author: ```suggestion
> - The `System.Drawing` namespace is not recommended for new development, because it's not supported within Windows services, ASP.NET Core, and ASP.NET. Attempting to use `System.Drawing` classes within one of these application types, may result in run-time exceptions and diminished service performance. Recommended alternatives include [ImageSharp](https://github.com/SixLabors/ImageSharp), [SkiaSharp](https://github.com/mono/SkiaSharp), and [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).
```

> [!CAUTION]

> > - The `System.Drawing` namespace is not recommended for new development, due to not being supported within a Windows or ASP.NET service. [ImageSharp](https://github.com/SixLabors/ImageSharp) and [SkiaSharp](https://github.com/mono/SkiaSharp) are recommended as alternatives.

Reviewer: ```suggestion
> - The `System.Drawing` namespace is not recommended for new development, because it's not supported within Windows services, ASP.NET Core services, and ASP.NET services. If you attempt to use `System.Drawing` classes within one of these application types, you may get run-time exceptions and diminished service performance. Recommended alternatives include [ImageSharp](https://github.com/SixLabors/ImageSharp), [SkiaSharp](https://github.com/mono/SkiaSharp), and [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).
```

> - On Windows, `System.Drawing` depends on the GDI+ native library, which is shipped as part of the OS. Some Windows SKUs, like Windows Server Core or Windows Nano, don't include this native library as part of the OS. Exceptions will be thrown at run time because the library can't be loaded.

> > - Classes within the <xref:System.Drawing> namespace are not supported for use within a Windows, ASP.NET Core, or ASP.NET service. Attempting to use these classes from within one of these application types may produce unexpected problems, such as diminished service performance and run-time exceptions. For a supported alternative, see [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).

Reviewer: ```suggestion
```
